packages/core/src/utilities/text-color/text-color.example.html

```diff
@@ -0,0 +1,30 @@
+<section class="example--list">
+  <% var groups = { 'background': ['base', 'black', 'gray', 'gray-dark', 'primary',
+  'primary-darker', 'primary-darkest', 'primary-alt-darkest', 'error', 'error-dark',
+  'error-darkest', 'success'], 'background-inverse': ['white','gray-lightest', 'gray-lighter',
+  'gray-light', 'primary-alt', 'primary-alt-light', 'primary-alt-lightest', 'primary-alt-dark'],
+  'gray-lightest': ['gray', 'gray-dark', 'base', 'primary', 'primary-darker',
+  'primary-darkest','primary-alt-darkest', 'error-dark', 'error-darkest'], 'primary-alt-lightest':
+  ['base', 'gray', 'gray-dark', 'primary-darker', 'primary-darkest', 'primary-alt-darkest',
+  'error-dark', 'error-darkest' ], 'success-lightest': ['base', 'gray', 'gray-dark', 'primary',
+  'primary-darker', 'primary-darkest', 'primary-alt-darkest', 'error-dark', 'error-darkest' ],
+  'warn-lightest': ['base', 'gray', 'gray-dark', 'primary', 'primary-darker', 'primary-darkest',
+  'primary-alt-darkest', 'error-dark', 'error-darkest' ], 'error-lightest': ['base', 'gray',
+  'gray-dark', 'primary-darker', 'primary-darkest', 'primary-alt-darkest', 'error-darkest' ], }; -%>
+  <% Object.keys(groups).forEach(fill => { -%>
+  <article>
+    <h2 class="preview__label ds-u-margin-top--2 ds-u-margin-bottom--4">
```

Comment:
This is a style preference, but I think this padding could also work:
`<h2 class="preview__label ds-u-margin-bottom--2">`
<img width="798" alt="Screen Shot 2019-12-19 at 11 22 21 AM" src="https://user-images.githubusercontent.com/5424713/71194580-d5082e80-2251-11ea-82aa-35acd014cecc.png">